### PR TITLE
Scripts

### PR DESCRIPTION
**Pull request by MichaelKim**
- Add scripts folder
  - Run script server separately
- Add weather, micro scripts
  - Implemented writing, reading files
- Smooth scrolling on terminals

### CODE REVIEW

File: build/main.css [hunk 1]
```diff
@@ -16,6 +16,7 @@ html, body, #root {
     width: 100%;
     height: 100%;
     padding: 20px;
+    overflow: scroll;
```
Comment (fg123) [inline build/main.css:19]: is this line necessary? does `auto` not work?

Comment (MichaelKim) [reply build/main.css:19]: yes otherwise the terminal can't scroll

File: package.json [hunk 2]
```diff
@@ -18,6 +19,8 @@
     "babel-loader": "^7.1.1",
     "babel-preset-env": "^1.6.0",
     "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-2": "^6.24.1",
+    "express": "^4.15.4",
```
Comment (fg123) [inline package.json]: Not required anymore

File: src/containers/terminal-link.jsx [hunk 3]
```diff
@@ -30,13 +39,35 @@ class TerminalLink extends React.Component {
         }
     }
 
+    startSmoothScroll(time, interpolator) {
```
Comment (fg123) [inline src/containers/terminal-link.jsx:54]: Great animation code!

**Review by fg123 — APPROVED** [review]

**Comment by fg123** [comment]
LGTM 👍

**Pull request merged**